src/software/physics/euclidean_to_wheel_test.cpp

```diff
@@ -8,130 +8,20 @@ class EuclideanToWheelTest : public ::testing::Test
-
-    EXPECT_TRUE(TestUtil::equalWithinTolerance(
-        expected_wheel_speeds,
-        euclidean_to_four_wheel.getTargetWheelSpeeds(target_euclidean_velocity,
-                                                     current_wheel_speeds),
-        0.001));
-
-    // test -/backwards
-    current_wheel_speeds      = {0, 0, 0, 0};
-    target_euclidean_velocity = {0, -1, 0};
-    expected_wheel_speeds     = {-2.1226, 2.1226, 2.7766, -2.7766};
-
-    EXPECT_TRUE(TestUtil::equalWithinTolerance(
-        expected_wheel_speeds,
-        euclidean_to_four_wheel.getTargetWheelSpeeds(target_euclidean_velocity,
-                                                     current_wheel_speeds),
-        0.001));
-}
-
-TEST_F(EuclideanToWheelTest, test_target_wheel_speeds_w)
-{
-    // test +/forwards
-    current_wheel_speeds      = {0, 0, 0, 0};
-    target_euclidean_velocity = {0, 0, 1};
-    expected_wheel_speeds     = {0.0918, 0.0918, 0.0885, 0.0885};
-
-    EXPECT_TRUE(TestUtil::equalWithinTolerance(
-        expected_wheel_speeds,
-        euclidean_to_four_wheel.getTargetWheelSpeeds(target_euclidean_velocity,
-                                                     current_wheel_speeds),
-        0.001));
-
-    // test -/backwards
-    current_wheel_speeds      = {0, 0, 0, 0};
-    target_euclidean_velocity = {0, 0, -1};
-    expected_wheel_speeds     = {-0.0918, -0.0918, -0.0885, -0.0885};
-
-    EXPECT_TRUE(TestUtil::equalWithinTolerance(
-        expected_wheel_speeds,
-        euclidean_to_four_wheel.getTargetWheelSpeeds(target_euclidean_velocity,
-                                                     current_wheel_speeds),
-        0.001));
-}
-
-TEST_F(EuclideanToWheelTest, test_target_wheel_speeds_all)
-{
-    // test +/forwards
-    current_wheel_speeds      = {0, 0, 0, 0};
-    target_euclidean_velocity = {1, 1, 1};
-    expected_wheel_speeds     = {-0.0440, -4.2893, 1.3231, 6.8763};
-
-    EXPECT_TRUE(TestUtil::equalWithinTolerance(
-        expected_wheel_speeds,
-        euclidean_to_four_wheel.getTargetWheelSpeeds(target_euclidean_velocity,
-                                                     current_wheel_speeds),
-        0.001));
-}
-
-TEST_F(EuclideanToWheelTest, test_sanity_check_conversion_is_linear)
-{
-    current_wheel_speeds      = {1, 2, 3, 5};
-    target_euclidean_velocity = {3, 1, 5};
-
-    auto result = euclidean_to_four_wheel.getTargetWheelSpeeds(target_euclidean_velocity,
-                                                               current_wheel_speeds);
-
-    current_wheel_speeds      = {100, 200, 300, 500};
-    target_euclidean_velocity = {300, 100, 500};
-
-    auto scaled_result = euclidean_to_four_wheel.getTargetWheelSpeeds(
-        target_euclidean_velocity, current_wheel_speeds);
-
-    EXPECT_TRUE(TestUtil::equalWithinTolerance(result * 100, scaled_result, 0.001));
+        euclidean_to_four_wheel.getWheelVelocity(target_euclidean_velocity), 0.001));
 }
```

Comment:
why're we deleting all these tests?

Reply:
I've rewritten pretty much all of these tests and will include it in a future PR once Im more comfortable with the conversion. I mainly rewrote the tests as they were too tightly coupled with the wheel angle.